### ISSUE TITLE
Eshell: changing shell-default-height take no effect?

### ISSUE THREAD

**Issue by microamp** [syl20bnr/spacemacs#4786]
Even when I increase the value of `shell-default-height`, the height of the shell stays the same; `30` which is the default value.

From my `.spacemacs/init.el`,
```emacs-lisp
     (shell :variables
            shell-default-height 66
            shell-default-position 'bottom)
```

`C-h v shell-default-height` shows me,
```
shell-default-height is a variable defined in `config.el'.
Its value is 66

Documentation:
Height in percents for the shell window.

[back]
```

which appears to be correct except that it's still using the default height value it seems.

**Comment by duerrp** [comment]
Try `shell-pop-window-size` instead...

**Comment by microamp** [comment]
> Try `shell-pop-window-size` instead...

Thanks. `shell-pop-window-size` does work for me. However, `layers/shell/README.org` [1] suggests otherwise. I guess either the doc or the actual code needs to be changed accordingly, provided that this is a bug.

Notes:
1. 
> It is possible to choose where the shell should pop up by setting the variable `shell-default-position` to either `top`, `bottom` or `full`. It is not possible to show it on the side for now. Default value is `bottom`. It is also possible to set the default height in percents with the variable `shell-default-height`. Default value is `30`.

**Comment by TheBB** [comment]
The Spacemacs code sets `shell-pop-window-height`, which is an alias of `shell-pop-window-size`. If you inspect the value of this/these variable(s) in the two different cases, are they different?

**Comment by microamp** [comment]
Updating `shell-pop-window-size` updates both variables as expected. However, updating `shell-default-height` (as suggested in docs) only updates the variable itself; `shell-pop-window-size` still shows the default value which is `30`.

**Comment by TheBB** [comment]
Thanks. Turns out this is actually a bug. I think https://github.com/syl20bnr/spacemacs/pull/4828 should fix it.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
